script.js

```diff
@@ -136,3 +137,17 @@ for (let i = 0; i < 6; i += 1) {
     });
   });
 }
+
+// FORM VALIDATION ////////////////////////////////////////////////////////////////////////////////
+const submit = document.querySelector('#submit');
+const errorMessage = document.querySelector('#error-message');
+submit.addEventListener('click', (event) => {
+  const emailInput = document.querySelector('#email');
+  const emailValue = emailInput.value;
+  if (emailValue !== emailValue.toLowerCase()) {
+    event.preventDefault();
+    errorMessage.innerHTML = 'Please use only lowercase email addresses!';
```

Comment:
[OPTIONAL] Well done on the validation 💯 but in order to improve it you can look to improve the styling, this is what you have so far 👇🏽 

![Screenshot 2023-03-10 at 22 34 18](https://user-images.githubusercontent.com/45739963/224462814-bcc05d8d-b904-4a7a-b566-89a4ce01fbbf.png)

You could maybe try red since it's an error ❌ and as a bonus, you could add a `setTimeout` to make it disappear a bit after the user has seen it (we don't need that error to stick around) 😅